api/nodes/serializers.py

```diff
@@ -248,6 +248,7 @@ class NodeSerializer(TaxonomizableSerializerMixin, JSONAPISerializer):
         'registrations',
         'contributors',
         'bibliographic_contributors',
+        'implicit_contributors',
```

Comment:
Assume we also want to hide this for anonymous vol's - let me know if this is an incorrect assumption.